### PR TITLE
Refactors json-ld validation.

### PR DESCRIPTION
**Pull request by justinlittman**
## Why was this change made?
* More idiomatic.
* DRY testing.

## How was this change tested?
Unit

### CODE REVIEW

**Comment by justinlittman** [comment]
@ndushay After re-reading the Express docs, I realized that there is a more idiomatic way to do this.

File: src/rdf.js [hunk 1]
```diff
@@ -43,3 +44,26 @@ export const n3FromJsonld = (jsonld, asTurtle) => {
 export const ttlFromJsonld = (jsonld) => {
   return n3FromJsonld(jsonld, true)
 }
+
+export const validateJsonld = async (req, resp, next) => {
```
Comment (ndushay) [inline src/rdf.js:48]: "validate" implies we are checking for "valid" jsonld rather than just parseable.  Maybe add a jsdoc comment?

**Review by ndushay — APPROVED** [review]
Much more elegant - I like it!

File: src/rdf.js (hunk 1 above)
Comment (justinlittman) [reply src/rdf.js:48]: Agree. I'm going to rename.

**Pull request merged**